.github/workflows/autofix.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
         timeout-minutes: 15
         env:
             CARGO_INCREMENTAL: false
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You could also put this at the top of the file in an `env` section (See `cpp_package.yaml`).